src/auth.ts

```diff
@@ -0,0 +1,33 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+function parseEnvironmentFile(src: string) {
+  // Try parse JSON
+  try {
+    return JSON.parse(src.toString());
```

Comment:
@gahabeen I'm assuming you added this in case the `src` that comes in is a stringify-ed JSON object. If we're going to be using a `.faunarc` instead of a `fauna.config.json` file, do we even need this?

@BrunoQuaresma Thoughts?

Reply:
Indeed. Just in case if the file is formatted in json format. Can be removed if we want to be strict about the format.

We could also allow support for `.faunarc` as well as `.faunarc.json`. Thus the `parseEnvironmentFile` (which could actually be renamed `parseConfigFile`) could be used for any of them.

Reply:
@gahabeen Got it 👍 Smart.

@BrunoQuaresma Do you want to support both files or be strict about enforcing only the `.faunarc`?

Reply:
I think be strict could be better to avoid confusion.